src/components/MdeEditor.tsx

```diff
@@ -17,41 +17,53 @@ export class MdeEditor extends React.Component<MdeEditorProps, {}> {
     handleOnChange = (editorState: EditorState) => {
         const {onChange} = this.props;
         onChange(editorState);
-    }
+    };
+
+    executeCastAsEditorState = (commandToExecute: Command, editorState: EditorState, data?: any): EditorState => {
```

Comment:
Javascript being annoying in general, had to typecast to be an object return rather than a Promise wrapping the EditorState